pyOCD/utility/conversion.py

```diff
@@ -18,18 +18,18 @@
 import struct
 import binascii
 
-## @brief Convert a byte array into a word array.
-def byte2word(data):
+## @brief Convert a list of bytes to a list of 32-bit integers (little endian)
+def byteListToLittleEndianU32List(data):
```

Comment:
Any reason this function has "LittleEndian" spelled out, whereas the others use "le"?